src/main/java/com/teachmeskills/employee_info/EmployeeInfoApplication.java

```diff
@@ -0,0 +1,14 @@
+package com.teachmeskills.employee_info;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+
+@SpringBootApplication(scanBasePackages = "com.teachmeskills.employee_info")
```

Comment:
Do you need scanBasePackages?

Reply:
Trying solutions from StackOverflow, also will remove it soon.

Reply:
Fixed.

Reply:
It's good to take code from somewhere, but you should do it carefully)
And also understand each line of code, you are taken)